src/client/Microsoft.Identity.Client/Cache/CacheRefreshReason.cs

```diff
@@ -15,18 +17,31 @@ public enum CacheRefreshReason
         /// <summary>
         /// When the token request goes to the identity provider because force_refresh was set to true. Also occurs if WithClaims() is used.
         /// </summary>
+        [Obsolete("Use ForceRefresh or WithClaims instead.")]
         ForceRefreshOrClaims = 1,
         /// <summary>
+        /// When the token request goes to the identity provider because force_refresh was set to true.
+        /// </summary>
+        ForceRefresh = 2,
+        /// <summary>
         /// When the token request goes to the identity provider because no cached access token exists
         /// </summary>
-        NoCachedAccessToken = 2,
+        NoCachedAccessToken = 3,
```

Comment:
@neha-bhargava is [correct](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5188#discussion_r1995897832) that this change does not look right. The original topic of that conversation was to keep an alias on the SDK implementation WITHOUT altering any old values. We shall never alter old values.

Please take this into consideration in the upcoming mini specs.